format-specs/geoparquet.md

```diff
@@ -14,38 +18,73 @@ Geometry columns are stored using the `BYTE_ARRAY` parquet type. They are encode
 geoparquet files include additional metadata at two levels:
 
 1. File metadata indicating things like the version of this specification used
-2. Column (chunk) metadata with additional metadata for each geometry column
+2. Column metadata with additional metadata for each geometry column
+
+These are both stored under a "geo" key in the parquet metadata.
 
 ## File metadata
 
-All file-level metadata should be included under the "geoparquet" key in the parquet metadata.
+All file-level metadata should be included under the "geo" key in the parquet metadata.
 
 |     Field Name     |  Type  |                             Description                              |
 | ------------------ | ------ | -------------------------------------------------------------------- |
-| geoparquet_version | string | **REQUIRED** The version of the metadata standard used when writing. |
-| primary_column     | string | The name of the "primary" geometry column.                           |
+| version     | string | **REQUIRED** The version of the geoparquet metadata standard used when writing.   |
+| primary_column     | string | **REQUIRED** The name of the "primary" geometry column.                |
+| columns            | \[[Column Metadata](#column-metadata)\]  | **REQUIRED** Metadata about geometry columns. |
```

Comment:
I think this type should be something like `Map<key, [Colum Metadata](#column-metadata)>`, where the key is the name of the column in the table.